ChangeLog.d/add-client-side-identity-hint-callback.txt

```diff
@@ -0,0 +1,2 @@
+Features
+   * Add support for a client side identity hint callback.
```

Comment:
Thanks for adding this, but please provide more context, for example: "In (D)TLS, add client-side support for the PSK callback, allowing to select a key based on the identity hint sent by the server."

Reply:
fixed